commands/peers/commands.go

```diff
@@ -0,0 +1,40 @@
+// Package peercommands implements the peer management commands
+package peercommands
+
+import (
+	"github.com/gluster/glusterd2/rest"
+)
+
+// Command is a holding struct used to implement the GlusterD Command interface
+type Command struct {
+}
+
+// Routes returns command routes. Required for the Command interface.
+func (c *Command) Routes() rest.Routes {
+	return rest.Routes{
+		rest.Route{
+			Name:        "GetPeer",
+			Method:      "GET",
+			Pattern:     "/peers/{peerid}",
```

Comment:
How we implement the ID-to-Name translation (and vice-versa) is not important. But it is something we will be doing.

No one other than us is going to be accessing the information stored in the store directly. So we are free to use UUIDs as the key. Using UUIDs to be the key is nice as they are guaranteed not to clash.